devdox/app/services/git_tokens_service.py

```
@@ -199,3 +199,32 @@ async def add_git_token(self, user_claims:UserClaims, json_payload: GitLabelBase
+        self.label_store = label_store
+
+    @classmethod
+    def with_dependency(
+        cls,
+        label_store: Annotated[TortoiseGitLabelStore, Depends()],
+    ) -> "DeleteGitLabelService":
+        return cls(
+            label_store=label_store,
+        )
+
+    async def delete_by_git_label_id(self, user_claims: AuthenticatedUserDTO, git_label_id:uuid.UUID) -> int:
```

Flake8 (7.2.0)

[error] missing whitespace after ':'

(E231)